cogs/moveusers.py

```diff
@@ -3,11 +3,12 @@
 
 # Command that moves one user in a voice channel into another voice channel specified by the user
 class MoveUsers(commands.Cog):
-
+# Function dedicated to moving users from one voice channel to another
```

Comment:
This function is to move a specified user to a specified channel. It is not necessarily based on the game they are playing, server role, etc. because that's more for the `automove` functionality @stephenlee91 is making.